v2-migration/src/main/resources/META-INF/rewrite/change-enum-getters.yml

```diff
@@ -0,0 +1,378 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: software.amazon.awssdk.v2migration.EnumGettersToV2
+displayName: Change v1 enum getters to v2
+recipeList:
+  - org.openrewrite.java.ChangeMethodName:
+      methodPattern: software.amazon.awssdk.services.sqs.model.CreateQueueRequest attributes()
```

Comment:
I think we need to perform this recipe before ChangeSdkType because v2 POJOs actually have both `xxx()` and `xxxAsString()` and this will cause issues for customers who use Java v1 and Java v2 in their workspace. Can we add a test case for it?

Reply:
Good catch, updated to run before ChangeSdkType and change v1 classnames, added end-to-end tests with v2 pojos included